src/main/java/network/brightspots/rcv/StreamingCvrReader.java

```diff
@@ -213,11 +213,16 @@ private void cvrCell(int col, String cellData) {
       currentPrecinct = cellData;
     } else if (idColumnIndex != null && col == idColumnIndex) {
       currentSuppliedCvrId = cellData;
-    }
+    } else if (col >= firstVoteColumnIndex
+        && (config.isMaxRankingsSetToMaximum()
+            || col < firstVoteColumnIndex + config.getMaxRankingsAllowedWhenNotSetToMaximum())) {
```

Comment:
We don't throw here if this check fails because we rely on `runAdditionalValidations` to catch it later?

Reply:
This is based on the following discussion: https://github.com/BrightSpots/rcv/issues/808#issuecomment-2028944594

Essentially, it's valid to have data after the last max ranking.